mr/src/main/java/org/elasticsearch/hadoop/cfg/Settings.java

```diff
@@ -221,6 +221,11 @@ public String getSerializerValueWriterClassName() {
         return getProperty(ES_SERIALIZATION_WRITER_VALUE_CLASS);
     }
 
+    public Settings setSerializerValueWriterClassName(String className) {
```

Comment:
Since the ValueWriter doesn't seem like the right place to ignore the document source for delete operations, this won't be needed.